src/test/java/com/commercetools/sync/sdk2/products/utils/ProductSyncMockUtils.java

```diff
@@ -125,36 +154,42 @@ public class ProductSyncMockUtils {
    */
   public static ProductDraftBuilder createProductDraftBuilder(
       @Nonnull final String jsonResourcePath,
-      @Nonnull final ResourceIdentifiable<ProductType> productTypeReference) {
-    final ProductProjection productFromJson =
-        readObjectFromResource(jsonResourcePath, Product.class).toProjection(STAGED);
+      @Nonnull final ProductTypeResourceIdentifier productTypeReference) {
```

Comment:
I would name it as `productTypeResourceIdentifier` to avoid ambiguity

Reply:
[4a031f5](https://github.com/commercetools/commercetools-sync-java/pull/886/commits/4a031f5e7e0ae733a462c4a5db4aca4c3bb08b2f)